### DISCUSSION TITLE
Is WebView2 free to use

### DISCUSSION THREAD

**Discussion by feng0133** [MicrosoftEdge/WebView2Feedback#4757] [Q&A] [answered]
Is WebView2 free to use

**Comment by jamesbgibson** [comment]
I assume so.

**Reply by feng0133** [reply]
Where can I see the license

**Reply by victorhuangwq** [reply, marked as answer]
There are licenses that you have to agree upon to use as an app developer: https://developer.microsoft.com/en-us/microsoft-edge/webview2/?form=MA13LH.

Click on download for the different kind of runtime that you would like to use, and the license should appear

But as of the time I'm posting, there shouldn't be any costs associated with using WebView2 in your application
